bundles/org.openhab.binding.tibber/README.md

```diff
@@ -32,6 +32,7 @@ Tibber Default:
 | Hourly Consumption | Hourly Consumption (last/previous hour) | True      |
 | Hourly From        | Timestamp (hourly from)                 | True      |
 | Hourly To          | Timestamp (hourly to)                   | True      |
+| Tomorrow           | JSON array of tomorrow's prices         | True      |
```

Comment:
Consider if it's worth documenting in more details. Currently the channel type description contains an example which is not documented here.

Reply:
Added full example